keyboards/sepsepkbd/yubino_hara/keyboard.json

```diff
@@ -0,0 +1,110 @@
+{
+    "manufacturer": "t-ando",
+    "keyboard_name": "yubino_hara",
+    "maintainer": "nagmeal",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "rgblight": true,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "split": {
+        "enabled": true,
+        "soft_serial_pin": "GP0",
+        "transport": {
+            "sync": {
+                "matrix_state": true
+            }
+        }
+    },
+    "matrix_pins": {
+        "cols": ["GP1", "GP2", "GP3", "GP4", "GP5", "GP6", "GP7"],
+        "rows": ["GP29", "GP28", "GP27", "GP26"]
+    },
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0x7A93"
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP14"
+    },
+    "rgblight": {
+        "led_count": 44,
+        "split": true,
+        "split_count": [22, 22],
+        "sleep": true,
+        "default": {
+            "hue": 0,
+            "sat": 0,
+            "val": 50
+        }
+    },
+    "layouts": {
+        "LAYOUT_split_6x3_3_1": {
```

Comment:
More of a nitpick, but layout name should be (Rows)x(Columns).

```suggestion
        "LAYOUT_split_3x6_3_1": {
```

Though, the keymaps would need to be changed too.

Reply:
Thank you very much!
I will fix it right away!

Is the maximum number correct?
It will be 7x4, so that's what I'll do.

Reply:
> It will be 7x4, so that's what I'll do.

![](https://i.imgur.com/WtB6bMy.jpeg)

So the image from your readme, has 7 rows?

Reply:
Sorry for the delay.

> Given it doesnt match a community layout, I would suggest to rename it to just LAYOUT to avoid your confusion.

Thanks for the great suggestions.
I wrote 7 rows but wasn't sure if it was really correct.
I will change it to LAYOUT.